packages/shared/src/hooks/useConditionalFeature.ts

```diff
@@ -21,32 +16,19 @@ export const useConditionalFeature = <T extends JSONValue>({
   feature: Feature<T>;
   shouldEvaluate: boolean;
 }): UseConditionalFeature<T> => {
-  const { user } = useContext(AuthContext);
   const { growthbook } = useGrowthBookContext();
   const { ready } = useFeaturesReadyContext();
-  const queryKey = generateQueryKey(
-    RequestKey.Feature,
-    user,
-    feature.id,
-    shouldEvaluate,
-  );
+  const isPending = !(shouldEvaluate && ready);
+  let value: WidenPrimitives<T> = feature.defaultValue as WidenPrimitives<T>;
 
-  const { data: featureValue, isPending } = useQuery({
-    queryKey,
-    queryFn: () => {
-      if (!shouldEvaluate) {
-        return feature.defaultValue as WidenPrimitives<T>;
-      }
-      return growthbook
-        ? growthbook.getFeatureValue(feature.id, feature.defaultValue)
-        : (feature.defaultValue as WidenPrimitives<T>);
-    },
-    enabled: shouldEvaluate && ready,
-    ...disabledRefetch,
-  });
+  if (!isPending) {
+    value = growthbook
+      ? growthbook.getFeatureValue(feature.id, feature.defaultValue)
```

Comment:
Does getFeatureValue has any side-effects to calling it in each render? That would be the major change vs useQuery implementation.

Reply:
yeah I see in source it indeed is just that

Reply:
so are we cool?

Reply:
From my side yes